diff -crB amber22_src/AmberTools/src/parmed/versioneer.py amber22_update/AmberTools/src/parmed/versioneer.py
*** amber22_src/AmberTools/src/parmed/versioneer.py	2021-10-31 03:31:23.000000000 +0800
--- amber22_update/AmberTools/src/parmed/versioneer.py	2024-04-19 13:58:22.000000000 +0800
***************
*** 335,343 ****
      # configparser.NoOptionError (if it lacks "VCS="). See the docstring at
      # the top of versioneer.py for instructions on writing your setup.cfg .
      setup_cfg = os.path.join(root, "setup.cfg")
!     parser = configparser.SafeConfigParser()
      with open(setup_cfg, "r") as f:
!         parser.readfp(f)
      VCS = parser.get("versioneer", "VCS")  # mandatory
  
      def get(parser, name):
--- 335,343 ----
      # configparser.NoOptionError (if it lacks "VCS="). See the docstring at
      # the top of versioneer.py for instructions on writing your setup.cfg .
      setup_cfg = os.path.join(root, "setup.cfg")
!     parser = configparser.ConfigParser()
      with open(setup_cfg, "r") as f:
!         parser.read_file(f)
      VCS = parser.get("versioneer", "VCS")  # mandatory
  
      def get(parser, name):
